data/resistance intro.txt

```diff
@@ -0,0 +1,1184 @@
+			choice
+				`	"Are you alright?"`
+					goto alright
+				`	"Have you decided to help them?"`
+					goto helpspider
+			label alright
+			`	"I'm fine, it's just... These all made me feel kind of impotent as I went through them, is all."`
+				goto helpspider
+			label helpspider
+			`	She looks back at you, then picks up some papers and books of her own from a drawer.`
+			`	"Bring these to those spider aliens; they're copies of everything I've found out here. If they want more, well, I'm not migrating to live among stupidly advanced aliens. Not yet, anyway. So they'd need to come here and research for themselves."`
+			`	She thanks you for bringing her the information, and you two say goodbye as you head back to your ship.`
+				accept
+
+
+
+mission "House Bebliss 6"
+	name "Jumping Spiders"
+	description "Acquire a spare Jump Drive, and bring it to <destination>, where the House Bebliss representatives will have it secretly installed in one of their ships."
+	landing
+	to offer
+		has "House Bebliss 5: done"
+	source "Factory of Eblumab"
+	destination "Blubipad's Workshop"
+	on offer
+		conversation
+			`You once again land close to the Arachi estate, as you prepare to bring them Freya's findings and her response.`
+			`	"To work with us, acepted, has she, Captain?"`
+			`	You tell them about Freya's response to their own findings, and hand them the copies of her studies, as you explain she has decided to not leave Deneb.`
+			`	"Understandable, that is," one of them says, as they all look over her findings. Although it's all written in a language they don't understand, they don't seem to have a problem staring for several minutes at the schematics she drew there.`
+			`	Finally, they hand the data to an employee, to have it translated, then say, "If come to us, this Freya will not, head to her, we shall."`
+			choice
+				`	"How do you plan to do that?"`
+				`	"Do you have a Jump Drive?"`
+			`	"Confidential information, from the Heliarch our House can gather. To steal a Jump Drive, however, willing to risk it we are not," they explain.`
+			`	"Assist us, you must, Captain. a Jump Drive, to <destination>, you must bring. One of our ships, with it we will equip."`
+			`	"<payment>, you will receive."`
+			`	You're guided back to your ship, as they claim they will now wait for Freya's findings to be translated, and then analyze them.`
+				accept
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		outfit "Jump Drive" -1
+		payment 1500000
+		conversation
+			`You land in another private area, owned by House Bebliss, as they guide you with the coordinates.`
+			`	They remove the Jump Drive from your cargo, and prepare to install it on one of ther own, as they hand you <payment>.`
+			`	"Once ready, the ship is, in the main spaceport, meet you, we will," one of the representatives says.`
```

Comment:
You might want to clarify that there isn't a timer for this so players know they just have to click on the spaceport, not wait.